laws/src/main/scala/org/http4s/laws/discipline/ArbitraryInstances.scala

```diff
@@ -571,27 +571,16 @@ private[http4s] trait ArbitraryInstances {
       } yield `Transfer-Encoding`(codings)
     }
 
-  implicit val http4sTestingArbitraryForRawHeader: Arbitrary[Header.Raw] =
+  implicit val http4sTestingArbitraryForRawHeader: Arbitrary[v2.Header.Raw] =
     Arbitrary {
       for {
         token <- genToken
         value <- genFieldValue
-      } yield Header.Raw(CIString(token), value)
+      } yield v2.Header.Raw(CIString(token), value)
     }
 
-  implicit val http4sTestingArbitraryForHeader: Arbitrary[Header] =
-    Arbitrary {
-      oneOf(
-        getArbitrary[`Accept-Charset`],
-        getArbitrary[Allow],
-        getArbitrary[`Content-Length`],
-        getArbitrary[Date],
-        getArbitrary[Header.Raw]
-      )
```

Comment:
This is not living up to its full potential anyway, but it would be nice to regain something like this, that generates real headers.

Reply:
I am totally lost